### ISSUE TITLE
Typescript build error: Cannot find name 'Tap'.

### ISSUE THREAD

**Issue by eliraz-refael** [zerobias/effector#270]
**What is the current behavior:**
TSC is unable to compile effector due to an error:
`node_modules/effector/index.d.ts:239:5 - error TS2304: Cannot find name 'Tap'.`

**Please provide the steps to reproduce and if possible a minimal demo of the problem via https://effector.now.sh/try, https://codesandbox.io or similar**
The bug happens on build time using TSC as a compiler.
also any ts-loader will complain about this error if checking is enabled.

**What is the expected behavior:**
To have TSC compile a project with recent effector in it.

**Which versions of Effector packages, and which browser and OS are affected by this issue? Did this work in previous versions of Effector?:**
v20.8.0

**Comment by bloadvenro** [comment]
Hi, @eliraz-refael!

It would be great if you could show steps you performed.

Please, try the following steps:

```bash
$ mkdir ~/effector-tsc-problem

$ cd ~/effector-tsc-problem

$ yarn init -y

$ yarn add typescript effector

$ yarn tsc --init

$ cat <<EOF > index.ts                                                                                                                                                                     
import {createStore} from 'effector';
const store = createStore(1);
EOF

```

Then specify `lib` and `files` section in your `tsconfig.json`:

```json
{
  "compilerOptions": {
    ...
    "lib": ["es6"],
    ...
  },
  "files": [
    "./index.ts"
  ]
}
``` 

Now, just issue `tsc` in your terminal without specifying any files or arguments.

Please report about your result.

**Comment by bloadvenro** [comment]
- `lib` option will deal with `Sat/Tap` errors
- either `files` or `include` sectons should always be specified as running `tsc file.ts` is the incorrect way of TS compilation

**Comment by zerobias** [comment]
This was fixed in [20.8.1](https://github.com/zerobias/effector/commit/22ef580c9426d778d2c70e6084d1fdfc6cb57184), please consider update
